lib/beaker-hostgenerator/hypervisor/abs.rb

```diff
@@ -37,12 +37,12 @@ def generate_node(node_info, base_config, bhg_version)
                  when 'AARCH64'
                    'arm64'
                  when 'POWER'
-                   base_template = node_info['ostype'].sub(/ubuntu(\d\d)/, 'ubuntu-\1.')
+                   base_template.sub!(/ubuntu-(\d\d)/, 'ubuntu-\1.')
```

Comment:
Why this change? It looks unrelated.

Reply:
I wanted to make things more similar with the vmpooler generator.

About this line I noticed we ran a sub on `node_info['ostype']` on line 31 to add a dash which is needed for all Ubuntu templates.

This line should just add a dot between the major and minor versions of Ubuntu (1804 -> 18.04) when the arch is POWER. I guess we can document this a bit though.

Reply:
My point is that in general it's a good practice to separate your changes to multiple commits.

Reply:
Yeah, I usually start with something simple and then tinker with stuff more and more lol

I'll update the commit messages to reflect these changes.